amundsen_application/static/css/_variables-default.scss

```diff
@@ -56,12 +56,20 @@ $line-height-large: 1.5 !default;
 
 // Badges
 $badge-text-color: $text-primary;
-$badge-danger-color: $sunset20;
-$badge-default-color: $gray20;
+$badge-negative-color: $sunset20;
```

Comment:
Last thing, we do have some places where we have hardcoded `labelStyle: 'danger'`, vs using `BadgeStyle.DANGER`.  If you search the `components` folder repo for `'danger'`, `'success'`, and `'default'` you will see a few places where we have to use the proper method of using `BadgeStyle`.